test.php

```diff
@@ -0,0 +1,8 @@
+<?php 
+
+echo "testing1:" . time() . PHP_EOL;
```

Comment:
:no_entry_sign: **Error( severity 5 )**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found &#039;time&#039; (*WordPress.Security.EscapeOutput.OutputNotEscaped*).